salt/apps/ocw/sync_repo.sls

```diff
@@ -1,9 +1,8 @@
-{% set github_ssh_key = salt.pillar.get('ocw:github_ssh_key') %}
-
 add_private_github_ssh_key:
   file.managed:
     - name: /root/.ssh/ocw_ssh_key
-    - contents_pillar: {{ github_ssh_key }}
+    - contents_pillar: |
+        {{ salt.pillar.get('ocw:github_ssh_key')|yaml(False)|indent(8) }}
```

Comment:
I just noticed that thsi is using `contents_pillar`, which means that there's no need to use Jinja at all. You just need to pass the key path, so in this case it should read `contents_pillar: ocw:github_ssk_key`